### DISCUSSION TITLE
Install without Jupyter?

### DISCUSSION THREAD

**Discussion by adam-mrozik** [great-expectations/great_expectations#6316] [Support]
Hey! Is there an option to install GE without Jupyter-like dependencies? They are deemed insecure for PROD images

**Comment by michalc** [comment]
This would be great for our use case for very similar reasons.

We mostly would like to run expectations in automatically run ETL pipelines where there is no interactivity, and so don't need to introduce the dependencies that can be subject to extra vulnerability alerts (that would not be justified because they don't be used in that context).

Also would mean install is just that tiny bit slower, so slows down the development of the pipelines, albeit by just a bit

And means we run the risk of getting into dependency hell a bit - Jupyter and its dependencies could conflict with our other dependencies. We might be able to sort this in some cases and separate things out, but maybe not all (e.g. in the case where we want failing checks to result in no change to our production data, and so be very intertwined with the ETL process)
